mobile/src/bundles/talent/components/profile-preview/profile-preview.tsx

```diff
@@ -37,6 +37,8 @@ const ProfilePreview: React.FC = () => {
         description,
     } = onboardingData;
 
+    const experienceYearsToDisplay = experienceYears && +experienceYears;
```

Comment:
why do we need this?

Reply:
To display numbers like 1 or 0.5, the variable "experienceYears" is a string, although the editor shows it as a number. I'm not sure why.